enhancements/must-gather-operator-ftpHost.md

```diff
@@ -0,0 +1,250 @@
+---
+title: must-gather-operator-ftpHost
+authors:
+  - "@shivprakahsmuley"
+reviewers:
+  - "@TrilokGeer"
+  - "@Prashanth684"
+approvers:
+  - "@TrilokGeer"
+  - "@Prashanth684"
+api-approvers:
+  - "@TrilokGeer"
+  - "@Prashanth684"
+creation-date: 2025-08-18
+last-updated: 2025-08-18
+tracking-link:
+  - https://issues.redhat.com/browse/MG-53
+status: implementable
+see-also:
+
+---
+
+# Must Gather Operator Enhancement: FTPHost in MustGatherSpec
+
+## Release Signoff Checklist
+
+- [ ] Enhancement is `implementable`
+- [ ] Design details are appropriately documented from clear requirements
+- [ ] Test plan is defined
+- [ ] Operational readiness criteria is defined
+- [ ] Graduation criteria for dev preview, tech preview, GA
+- [ ] User-facing documentation is created in [openshift-docs](https://github.com/openshift/openshift-docs/)
+
+## Summary
+
+Add an optional field `ftpHost` to `MustGather.spec` that allows directing must-gather artifacts to a designated FTP/SFTP endpoint during the upload phase. This provides a supported, secure, and validated path for environments that require alternate upload targets beyond the default Red Hat case management destination.
+
+Default: `access.redhat.com` (used when ftpHost is unset)
+
+## Motivation
+
+### User Stories
+
+* As a cluster admin, I can set `spec.ftpHost: access.stage.redhat.com` to stage bundles to pre‑production.
+* As a support engineer, I can ensure bundles always upload to the default `access.redhat.com` without specifying anything in the CR.
+* As a CI/QA engineer, I want to direct must-gather uploads to a non-production staging environment to avoid polluting production case management data.
+
+### Goals
+
+* Add `spec.ftpHost` with a safe default.
+* Ensure the upload container honors ftpHost and existing proxy settings.
+* Maintain full backward compatibility.
```

Comment:
To attain this goal (in the future), should we adhere with adding `spec.uploadTargets`?

ref: https://github.com/openshift/enhancements/pull/1830#discussion_r2378893089

Reply:
yes, this should reflect the same change in #1830 so we are unified on the API changes